gvr-javascript/app/src/main/java/org/gearvrf/sample/gvrjavascript/GearVRJavascriptActivity.java

```diff
@@ -59,7 +60,7 @@ public void onCreate(Bundle savedInstanceState) {
                 // Attach a script file directly (without using a GVRScriptBundle)
                 GVRScriptFile scriptFile;
                 try {
-                    scriptFile = sm.loadScript(
+                    scriptFile = (GVRScriptFile)sm.loadScript(
```

Comment:
Why the cast? Just use IScriptFile instead of GVRScriptFile.

Reply:
Will do